uint64 msg_id
uint8 window_size
sensor_msgs/Image image
float64[16] camToWorld
float64[4] Intrinsics
